install.sh

```diff
@@ -80,7 +80,7 @@ while true; do
     esac
     done
 
-rocm_path=/opt/rocm/bin
+rocm_path=$install_prefix/bin
```

Comment:
This isn't quite correct.  If the user decides to install to a new location (other than where a ROCm installation exists), this will fail because no compiler will be there.

Reply:
Is this install.sh used in the current form?

Reply:
Yes

Reply:
I fail to see how this has been resolved.

Reply:
I think it has, that change is removed. Only one line I see

Reply:
Yes, After looking at it closely, that change was not relevant for this PR and removed it.
Will submit another PR with a proper fix if that is needed.

Reply:
Ah - my mistake.  I clicked on view changes and saw the outdated changes.